### ISSUE TITLE
bone mineral density of humerus, vertebrae

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3638]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. decreased bone mineral density of humerus
2. decreased bone mineral density of vertebrae
3. decreased bone mineral density of lumbar vertebrae
4. increased bone mineral density of humerus
5. increased bone mineral density of vertebrae
6. increased bone mineral density of lumbar vertebrae

>2. Synonyms (e.g., absent spleen)

1. decreased BMD of humerus; decreased humerus bone mineral density
2. decreased BMD of vertebrae; decreased vertebrae bone mineral density
3. decreased BMD of lumbar vertebrae; decreased lumbar vertebrae bone mineral density
4.  increased BMD of humerus;   increased humerus bone mineral density 
5. increased BMD of vertebrae;   increased vertebrae bone mineral density 
6. increased BMD of lumbar vertebrae;   increased lumbar vertebrae bone mineral density

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. reduction in the quantitative measurement value of mineral content of bone in the upper bone of the foreleg
2. reduction in the quantitative measurement value of mineral content of bone in any of the bones of the spine
3. reduction in the quantitative measurement value of mineral content of bone in the lumbar vertebrae of the spine
4. elevation in the quantitative measurement value of mineral content of bone in the upper bone of the foreleg
5. elevation in the quantitative measurement value of mineral content of bone in any of the bones of the spine
6. elevation in the quantitative measurement value of mineral content of bone in the lumbar vertebrae of the spine

PMID:22446865

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. decreased bone mineral density (MP:0000063)
2. decreased bone mineral density (MP:0000063)
3. decreased bone mineral density of vertebrae (new term)
4. increased bone mineral density (MP:0000062)
5. increased bone mineral density (MP:0000062)
6. increased bone mineral density of vertebrae (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
Added terms 
MP:0021184 | decreased bone mineral density of humerus
MP:0021185 | increased bone mineral density of humerus
MP:0021186 | decreased bone mineral density of vertebrae
MP:0021187 | increased bone mineral density of vertebrae
MP:0021188 | decreased bone mineral density of presacral vertebrae
MP:0021189 | increased bone mineral density of presacral vertebrae
MP:0021190 | decreased bone mineral density of lumbar vertebrae
MP:0021191 | increased bone mineral density of lumbar vertebrae
MP:0021192 | decreased bone mineral density of sacral vertebrae
MP:0021193 | increased bone mineral density of sacral vertebrae
MP:0021194 | decreased bone mineral density of caudal vertebrae
MP:0021195 | increased bone mineral density of caudal vertebrae
MP:0021196 | decreased bone mineral density of cervical vertebrae
MP:0021197 | increased bone mineral density of cervical vertebrae
